keyboards/signum31/signum31.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include "quantum.h"
+
+#define LAYOUT( \
+	K000, K001, K002, K003, K004, K005, K006, K007, K008, K009, \
```

Comment:
Lines 6-11 should be edited to resemble the physical layout of the assembled keyboard, and the layer blocks...

https://github.com/qmk/qmk_firmware/blob/f4750687bda728ec486496f89c14ebf54d177c0e/keyboards/signum31/keymaps/default/keymap.c#L643-L649

... of the `keymap.c` files should be edited accordingly.